shared/src/persistence/elasticsearch/helpers/formatMessageResult.js

```diff
@@ -0,0 +1,32 @@
+const AWS = require('aws-sdk');
+
+exports.formatMessageResult = ({ caseMap, hit, sourceUnmarshalled }) => {
+  const casePk = hit['_id'].split('_')[0];
+  const docketNumber = casePk.replace('case|', '');
+
+  let foundCase = caseMap[docketNumber];
+
+  if (!foundCase) {
+    hit.inner_hits['case-mappings'].hits.hits.some(innerHit => {
+      const innerHitDocketNumber = innerHit['_source'].docketNumber.S;
+      caseMap[innerHitDocketNumber] = innerHit['_source'];
+
+      if (innerHitDocketNumber === docketNumber) {
+        foundCase = innerHit['_source'];
+        return true;
+      }
+    });
+  }
+
+  if (foundCase) {
+    const foundCaseUnmarshalled = AWS.DynamoDB.Converter.unmarshall(foundCase);
+
+    return {
+      leadDocketNumber: foundCaseUnmarshalled.leadDocketNumber,
```

Comment:
why do we do this?  shouldn't the ...foundCaseUnmarshalled directly below already have that property?